docs/tracking.md

```diff
@@ -0,0 +1,192 @@
+    |           `-- $KERMIT
+    |               |-- heads
+    |               |   `-- muppet-mania
+    |               `-- rad
+    |                   |-- id # <-- points to the identity of $KERMIT_IDENTITY
+    |                   |-- signed_refs
+    |                   |-- self
+    |                   `-- ids
+    |                       `-- $KERMIT_IDENTITY
+    |-- $KERMIT_IDENTITY
+    |   `-- refs
+    |       |-- heads
+    |       `-- rad
+    |           |-- id
+    |           |-- signed_refs
+    |           |-- self
+    |           `-- ids
+    |               `-- $KERMIT_IDENTITY
+    `-- $MISS_PIGGY_IDENTITY
+        `-- refs
+            |-- heads
+            `-- rad
+                |-- id
+                |-- signed_refs
+                |-- self
+                `-- ids
+                    `-- $MISS_PIGGY_IDENTITY
+```
+
+Again, `$MISS_PIGGY` and `$MISS_PIGGY_IDENTITY` are variables for Miss Piggy's peer identifier for
+the device she's using and the her identity document.
+
+As well as this entry of `remotes/$KERMIT`, Miss Piggy's monorepo will also gain an entry in its
+`config` file:
+
+```
+[remote "$THE_MUPPET_SHOW/$KERMIT"]
+        url = rad-p2p://$MISS_PIGGY@$KERMIT/$THE_MUPPET_SHOW.git?
+        fetch = +refs/heads/*:refs/remotes/$KERMIT/$THE_MUPPET_SHOW/*
+```
+
+The fetch refspec[5] here will seem off to you when comparing it to the above layout of a single
+monorepo, and you would be right. In this case Link only cares about the `url` and manages the fetch
+refspecs under the hood. The URL allows use to manage who we are tracking and for what URN.
+
+What we are learning from this exposition is that we add a peer as someone we track when we clone[3]
+their project. Miss Piggy is adding Kermit as a `remote`, and so she is tracking his changes. If
+Kermit announces changes and Miss Piggy's peer hears about these changes -- whether through Kermit's
+device or some other peer -- she will apply them to update her view of Kermit's remote.
+
+### Directly Track
+
+The other way a peer can track is by explicitly knowing a peer's identifier and telling their
+monorepo that they are interested in this peer. For example, Miss Piggy also heard that Rizzo the
+Rat is also taking part in the project and knows his peer identifier, $RIZZO. When she uses the
+`track`[4] function it creates a new entry in the monorepo config:
+
+```
+[remote "$THE_MUPPET_SHOW/$RIZZO"]
+        url = rad-p2p://$MISS_PIGGY@$RIZZO/$THE_MUPPET_SHOW.git?
+        fetch = +refs/heads/*:refs/remotes/$RIZZO/$THE_MUPPET_SHOW/*
+```
+
+The next time she hears from the network about changes from Rizzo she will fetch those changes and
+apply them to her monorepo.
+
+## The Tracking Exchange
+
+In the above example some extra details were elided because we were only talking about two peers. So
+when Miss Piggy cloned the repository from Kermit, it was almost obvious that he would end up in the
+`remotes`. But what if we talked about more peers?
```

Comment:
```suggestion
`remotes`. But what if we talked to more peers?
```

Is this a typo?

Reply:
Nope, I'm referring to "What if we added more peers?" to our story. I can try to make it more clear :)